src/views/confirmation-member-added.njk

```diff
@@ -3,35 +3,39 @@
 
 {% extends "layouts/default.njk" %}
 
-{% set userDetails = newUserDetails.userName if newUserDetails.userName else newUserDetails.email %}
-{% set roleText = lang.as_a + userRole if userRole == lang.standard_user else lang.as_an + userRole%}
+{% set userDetails = newUserDetails.userName if newUserDetails.userName else 
+    newUserDetails.email %}
+{% set roleText = lang.as_a + userRole if (userRole | lower === lang.standard_user | lower) 
```

Comment:
Looks good but it was a little difficult to see these were the only "relevant" changes as a lot of the other indentation on the file was changed.

Reply:
Ah, yes, sorry for that. My VS Code is configured to automatically reformat code on save and it applies the formatting to the whole file.